test/types.d.ts

```diff
@@ -48,7 +48,7 @@ interface TestSetup<STATE> {
     MAC: boolean;
     LINUX: boolean;
     WIN: boolean;
-    playwright: import('../src/server/browserType').BrowserType;
+    playwright: import('../src/server/browserType').BrowserType<import('../src/browser').Browser>;
```

Comment:
Let's migrate this whole file to public types in a follow up.

Reply:
Definitely!